gui/wxpython/docs/wxgui_sphinx/conf.py

```diff
@@ -25,8 +27,6 @@
     0, os.path.abspath(os.path.join(os.environ["GISBASE"], "etc", "python", "grass"))
 )
 
-from grass.script import core
-
```

Comment:
This might not behave successfully. Is there a way to know if it imports well, and not only wrapped inside a `grass --exec python`. I've hit some long import chains that needed everything already up and running in order to have something similar working. To confirm.

Reply:
I ran GRASS GIS, accessed the interactive terminal, entered the Python shell within that environment, and executed the import statement without encountering any errors.

Reply:
In fact, I see that this file is for generating docs (sphinx). Does someone know how to trigger it to check it?